app/component/map/MapBottomsheetContext.js

```diff
@@ -1,5 +1,3 @@
-import React from 'react';
+import { createContext } from 'react';
 
-const context = React.createContext({ paddingBottomRight: [0, 0] });
-
-export default context;
+export default createContext(0);
```

Comment:
Maybe this needs a comment to clarify meaning to this number, since without it this might seem little cryptic.